lib/collection/src/grouping/types.rs

```diff
@@ -32,7 +32,7 @@ impl Group {
 impl From<Group> for PointGroup {
     fn from(group: Group) -> Self {
         Self {
-            hits: group.hits,
+            hits: group.hits.into_iter().map(Into::into).collect(),
```

Comment:
Let's use concrete type instead of `Into::into`

Reply:
fixed